tools/internal/config/config.go

```diff
@@ -145,32 +127,6 @@ func (config *Config) ToRegsyncConfig() (regsync.Config, error) {
 	return regsyncYaml, nil
 }
 
-// ToRegsyncImages converts image into one ConfigSync (i.e. an image
-// for regsync to sync) for each tag present in image. repo provides
-// the target repository for each ConfigSync.
-func (image *Image) ToRegsyncImages(repo Repository) ([]regsync.ConfigSync, error) {
-	entries := make([]regsync.ConfigSync, 0, len(image.Tags))
-	for _, tag := range image.Tags {
-		doNotMirror, err := image.DoNotMirrorTag(tag)
-		if err != nil {
-			return nil, fmt.Errorf("failed to determine whether tag %q should be mirrored: %w", tag, err)
-		}
-		if doNotMirror {
-			continue
-		}
-		sourceImage := image.SourceImage + ":" + tag
-		targetImage := repo.BaseUrl + "/" + image.TargetImageName() + ":" + tag
-		entry := regsync.ConfigSync{
-			Source: sourceImage,
-			Target: targetImage,
-			Type:   "image",
-		}
-		entries = append(entries, entry)
-	}
-
-	return entries, nil
-}
-
 func CompareImages(a, b *Image) int {
```

Comment:
Can we move this to the `image.go` file too?

Reply:
Funny enough, I originally moved it, but I moved it back. I don't feel strongly either way though - I'll move it to `image.go` as you suggest.